Makefile

```diff
@@ -0,0 +1,29 @@
+OPENTELEMETRY_CHART = ./charts/opentelemetry-collector
+EXAMPLES_DIRECTORY = ./examples/opentelemetry-collector
+EXAMPLES = $(notdir $(shell find $(EXAMPLES_DIRECTORY) -type d -maxdepth 1 -mindepth 1))
+TMP_DIRECTORY = ./tmp
+
+.PHONY: generate-examples
+generate-examples:
+	for example in $(EXAMPLES); do \
```

Comment:
We should add similar examples to the operator chart later. So it would be nice if this target is not restricted to `opentelemetry-collector` chart, but can generates manifests for both charts potentially.

Reply:
Good thinking.  I'll update it look for examples in operator also.